consensus/state_processing/src/block_replayer.rs

```diff
@@ -60,13 +60,13 @@ impl From<BlockProcessingError> for BlockReplayError {
     }
 }
 
-/// Defines how state roots should be computed during block replay.
-#[derive(PartialEq)]
-pub enum StateRootStrategy {
+/// Defines how state roots should be computed and whether to perform all state transitions during block replay.
+#[derive(PartialEq, Clone)]
```

Comment:
May as well make it `Copy`

```suggestion
#[derive(PartialEq, Clone, Copy)]
```

Reply:
Nice tip!! 🙏